tests/test_data/cookiesandcups.com/cookiesandcups_1.json

```diff
@@ -0,0 +1,56 @@
+{
+  "author": "Cookies &amp; Cups",
```

Comment:
Possible HTML entity decoding issue here; note also that sometimes this website provides a per-recipe author (e.g. Shelly credited for the second test case).